linux_os/guide/system/accounts/accounts-banners/banner_etc_issue/tests/banner_etc_issue_cis.fail.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+# profiles = xccdf_org.ssgproject.content_profile_cis
```

Comment:
I suggest you add also other profiles derived from the CIS control file.

Reply:
Hi @vojtapolasek , do you mean `server_l1`, `server_l2`, `workstation_l1`, and workstation_l2`? If so, are they necessary in this header since they are derivations of the main profile? Anyways, I am testing here and will include them since this could be helpful and doesn't look to have any down side. : )

Reply:
> Well it could be helpful if you want to test the whole profile, but not the cis server level 2 which is marked as cis in the content. So remaining profiles are cis_server_l1, cis_workstation_l1 and cis_workstation_l2.

I have tested the rule against these other profiles, just to see the behavior and indeed it is good to include these derivations. Thanks for rise this. : ) Sending an update soon.